lib/ansible/modules/files/file.py

```diff
@@ -181,6 +181,12 @@
     state: file
     modification_time: now
     access_time: now
+
+- name: Set access time based on seconds from epoch value
+  file:
+    path: /etc/another_file
+    state: file
+    access_time: '{{ "%Y%m%d%H%M.%S" | strftime(stat_var.stat.atime) }}'
```

Comment:
I don't think your strftime template is correct

Reply:
Hmm it worked for me, what part do you think isn't correct?

Reply:
i always used '%s' .. won't the above print yearmonthdayhourminute. seconds?

Reply:
@jborean93 i think you are doing the opposite transformation, you are turning 'seconds/epoch' into a date string